packages/gestalt/src/Masonry.js

```diff
@@ -5,19 +5,24 @@ import FetchItems from './FetchItems.js';
 import styles from './Masonry.css';
 import { type Cache } from './Masonry/Cache.js';
 import defaultLayout from './Masonry/defaultLayout.js';
+import defaultTwoColumnModuleLayout from './Masonry/defaultTwoColumnModuleLayout.js';
 import fullWidthLayout from './Masonry/fullWidthLayout.js';
+import HeightsStore, { type HeightsStoreInterface } from './Masonry/HeightsStore.js';
 import MeasureItems from './Masonry/MeasureItems.js';
 import MeasurementStore from './Masonry/MeasurementStore.js';
 import ScrollContainer from './Masonry/ScrollContainer.js';
 import { getElementHeight, getRelativeScrollTop, getScrollPos } from './Masonry/scrollUtils.js';
+import { type Position } from './Masonry/types.js';
 import uniformRowLayout from './Masonry/uniformRowLayout.js';
 import throttle, { type ThrottleReturn } from './throttle.js';
 
 const RESIZE_DEBOUNCE = 300;
 
-const layoutNumberToCssDimension = (n: ?number) => (n !== Infinity ? n : undefined);
+// When there's a 2-col item in the most recently fetched batch of items, we need to measure more items to ensure we have enough possible layouts to find a suitable one
+// The number of items measured at a time is the number of columns * this multiplier
+const TWO_COL_ITEMS_MEASURE_BATCH_SIZE = 6;
```

Comment:
maybe a comment around why this is static and why we're doing 6

Reply:
Ah good call, I didn't update that comment